[2022-04-10 02:59:59,031] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-04-10T02:59:55.157582+00:00 [queued]>
[2022-04-10 02:59:59,049] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.accurate manual__2022-04-10T02:59:55.157582+00:00 [queued]>
[2022-04-10 02:59:59,051] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-10 02:59:59,052] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-10 02:59:59,053] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-10 02:59:59,068] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): accurate> on 2022-04-10 02:59:55.157582+00:00
[2022-04-10 02:59:59,074] {standard_task_runner.py:52} INFO - Started process 3297 to run task
[2022-04-10 02:59:59,078] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'accurate', 'manual__2022-04-10T02:59:55.157582+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpckwato61', '--error-file', '/tmp/tmpmdd_viw3']
[2022-04-10 02:59:59,079] {standard_task_runner.py:80} INFO - Job 12: Subtask accurate
[2022-04-10 02:59:59,146] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.accurate manual__2022-04-10T02:59:55.157582+00:00 [running]> on host c0c6f6bf1c08
[2022-04-10 02:59:59,237] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-04-10T02:59:55.157582+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-10T02:59:55.157582+00:00
[2022-04-10 02:59:59,254] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-10 02:59:59,256] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2022-04-10 02:59:59,266] {subprocess.py:85} INFO - Output:
[2022-04-10 02:59:59,270] {subprocess.py:89} INFO - accurate
[2022-04-10 02:59:59,271] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-10 02:59:59,297] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=accurate, execution_date=20220410T025955, start_date=20220410T025959, end_date=20220410T025959
[2022-04-10 02:59:59,332] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-10 02:59:59,373] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
